iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/checks/AbstractLimitsCheckTest.java

```diff
@@ -62,20 +58,6 @@ void testGetFirstChildElement() {
     assertThat(firstChildElement).isNull();
   }
 
-  @Test
-  void shouldNotReportLimitIssueWhenProjectHasNoLimitRange() {
-    var projectContext = ProjectContext.builder().setLimitRange(Trilean.FALSE).build();
-    var checkContext = checkContextAfterReportMissingLimit(projectContext);
-    verify(checkContext).reportIssue(ABSENT_TEXT_RANGE, MESSAGE);
-  }
-
-  @Test
-  void shouldNotReportLimitIssueWhenProjectHasLimitRange() {
-    var projectContext = ProjectContext.builder().setLimitRange(Trilean.TRUE).build();
-    var checkContext = checkContextAfterReportMissingLimit(projectContext);
-    verify(checkContext, never()).reportIssue((TextRange) any(), any());
-  }
-
   CheckContext checkContextAfterReportMissingLimit(ProjectContext projectContext) {
```

Comment:
This method and the static object at the beginning are not used anymore. TestAbstractLimitsCheck most likely as well